src/oc/web/components/fullscreen_post.cljs

```diff
@@ -444,9 +547,22 @@
                                  :use-inline-media-picker false
                                  :multi-picker-container-selector "div#fullscreen-post-box-footer-multi-picker"})
               [:div.fullscreen-post-box-content-body.fs-hide
-                {:key (str "fullscreen-post-body-" (:updated-at activity-data))
+                {:key (str "fullscreen-post-body-" (:updated-at current-activity-data))
                  :ref :fullscreen-post-box-content-body
-                 :dangerouslySetInnerHTML (utils/emojify (:body activity-data))}])
+                 :dangerouslySetInnerHTML (utils/emojify (:body current-activity-data))}])
+            ; (when (and (:video-transcript current-activity-data)
```

Comment:
Add a comment since this is related to transcript data we want to use.  Or maybe a feature flag?